source/config/localhost_config.json

```diff
@@ -7,5 +7,10 @@
        "server_cert": "",
        "server_key": "",
        "root_cert": ""
+    },
+    "cpu_affinity_configuration": {
```

Comment:
The PR doesn't describe "why" we are doing this very well.

The pre-existing code already assigns the sideband loop to a single CPU so that it can be isolated and cache invalidations/refreshes are minimized in performance critical code.

 There are a few different features here:

1. User can decide *which* CPU to assign. Why? Because they have more/fewer CPUs? They have more dedicated tasks? We want them to think about it?
2. The default changes to "no affinity"
3. User can assign the CPU of the server and the non-sideband read/write threads. Why? For performance of those threads? Or to keep them away from the core that the sideband thread uses?

Reply:
Updated the PR description.